instructor/cli/jobs.py

```diff
@@ -99,11 +99,28 @@ def watch(
 def create_from_id(
     id: str = typer.Argument(..., help="ID of the existing fine-tuning job"),
     model: str = typer.Option("gpt-3.5-turbo", help="Model to use for fine-tuning"),
-):
+    hyperparameters: str = typer.Option(None, help="Hyperparameters for fine-tuning"),
```

Comment:
I think this is the last thing, can we add these options as seperate flags, then I'll merge.

<img width="644" alt="Screenshot 2023-11-06 at 3 35 56 PM" src="https://github.com/jxnl/instructor/assets/4852235/70ec06dd-b838-4419-a05f-d404d92508a3">